html/themes/custom/common_design_subtheme/components/rw-article/rw-article.css

```
@@ -21,15 +25,19 @@
   position: relative;
   display: flex;
   flex-direction: column;
-  padding: 24px 0;
-  border-top: 1px solid #e6ecef;
+  padding: 0 0 24px 0;
```

```suggestion
  padding: 0 0 var(--cd-flow-space) 0;
```

With this change the padding would have the same value as the margin on the other side of the border. What do you think?

---

I don't think there is a need to mix the function of the `cd-flow` here. Padding is about what happens within a component, the flow is about how the component interacts with other elements around it. I don't mind using variables for padding but I am against using the same variable because they are not the same thing.

---

Fair enough.